cmd/restic/global.go

```diff
@@ -355,7 +355,7 @@ func OpenRepository(opts GlobalOptions) (*repository.Repository, error) {
 		return nil, err
 	}
 
-	if stdoutIsTerminal() {
+	if stdoutIsTerminal() && !opts.JSON {
```

Comment:
Just realized I made a similar, but slightly different variant of this change over at #1962: https://github.com/restic/restic/pull/1962/files#diff-952c97fd56de92e641e61b3b3aa881dbR363